cmd/internal/collector/query.go

```diff
@@ -94,6 +95,17 @@ func BuildQuery(metric MetricDescription, clusters []string, groupByLabels []str
 		Filters: clusterFilters,
 	})
 
+	// Remove topics from the topicFiltering list if they also exist in the ExcludeTopics list
```

Comment:
I think we should add a control to block a rule to specify both the `topics` and `excludeTopics` option.
From a logical point of view, having inclusion and exclusion causes is contradictory.

Reply:
I like this idea much better actually. I'll have it fail if this is the case.